pom.xml

```diff
@@ -85,7 +85,12 @@
         <spark.compat.version>3.4</spark.compat.version>
         <spark.major.version>3</spark.major.version>
         <log4j.version>2.19.0</log4j.version>
-        <graphframe.version>0.8.3-spark3.4</graphframe.version>
+
+
+        <graphframes.groupId>io.graphframes</graphframes.groupId>
+        <graphframes.artifactId>graphframes-spark${spark.major.version}_${scala.compat.version}</graphframes.artifactId>
```

Comment:
Probably don't need these as variables anymore if we're using the same thing for all Spark versions now, can just inline this